WikiWatershed/model-my-watershed
# Wire up Papertrail

## Overview

This PR adds support for Papertrail based logging (minus modifications to CloudFormation).

Resolves #3104 

## Testing Instructions

- Apply the following diff:

```diff
diff --git a/deployment/ansible/roles/model-my-watershed.base/meta/main.yml b/deployment/ansible/roles/model-my-watershed.base/meta/main.yml
index 8e51f9fa..8295f81a 100644
--- a/deployment/ansible/roles/model-my-watershed.base/meta/main.yml
+++ b/deployment/ansible/roles/model-my-watershed.base/meta/main.yml
@@ -4,4 +4,4 @@ dependencies:
   - { role: "azavea.git" }
   - { role: "azavea.daemontools" }
   - { role: "azavea.postgresql-support" }
-  - { role: "azavea.papertrail", when: "['development'] | is_not_in(group_names)" }
+  - { role: "azavea.papertrail" }
diff --git a/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml b/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml
index 0247f3d6..f8ac36d9 100644
--- a/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml
+++ b/deployment/ansible/roles/model-my-watershed.base/tasks/configuration.yml
@@ -31,4 +31,3 @@
     - Restart Celery

 - import_tasks: papertrail.yml
-  when: "['development'] | is_not_in(group_names)"
```
- Run `./scripts/vagrant-up.sh`
- See: https://papertrailapp.com/groups/14907082/events
